scripts/generate_conda_file.sh

```diff
@@ -9,6 +9,8 @@
 # $ sh generate_conda_file.sh --pyspark
 # For generating a conda file for running python gpu and pyspark:
 # $ sh generate_conda_file.sh --gpu --pyspark
+# For generating a conda file for running python gpu and pyspark with a particular version:
```

Comment:
We had a discussion on this to support DSVM - we need to match the Spark version available on DSVM but it is updating every now and then. @maxkazmsft

Reply:
Do you think the default version to be the version current on DSVM?

Reply:
Hmmm yea because we encourage users in the repo to try out notebooks on a Linux DSVM where some tools/software (e.g., Spark) have been pre-installed.

Reply:
I think we should keep this PR as consistent with the prior version used (2.3.1), and then have an additional update that tests against a newer version of spark. Changing this code then also updating the spark version is doing two different things. @miguelgfierro - what do you think?

Reply:
resolving to take this approach after talking with @miguelgfierro off-line.